.storybook/polaris-readme-loader.js

```diff
@@ -222,6 +222,7 @@ import {
   ConversationMinor,
   DeleteMinor,
   CircleDisableMinor,
+  CircleTickOutlineMinor,
```

Comment:
This is going to require updating an extra icon to the list of imports in polaris-styleguide's stuff too. I'd like to keep avoid adding to these lists as it is a pain to remove things later. Any chance you could use another icon that's already on the blessed list?

Reply:
That seems like an issue with the styleguide. I'll fix it up though. Thanks Ben!